packages/vendor-core/package.json

```diff
@@ -26,11 +26,11 @@
   },
   "dependencies": {
     "@novnc/novnc": "novnc/noVNC#25b3d49d322b0a7c9ee1e071d93042d70f5176b7 ",
-    "@patternfly/patternfly": "^4.10.31",
-    "@patternfly/react-core": "^4.18.5",
-    "@patternfly/react-icons": "^4.3.5",
-    "@patternfly/react-table": "^4.5.7",
-    "@patternfly/react-tokens": "^4.4.4",
+    "@patternfly/patternfly": "^4.16.7",
+    "@patternfly/react-core": "^4.23.1",
+    "@patternfly/react-icons": "^4.4.2",
+    "@patternfly/react-table": "^4.8.6",
+    "@patternfly/react-tokens": "^4.5.2",
```

Comment:
Updated the pf to fix the snapshot issues with the component names: https://github.com/patternfly/patternfly-react/pull/4391